src/pages/careers/locations/_location-layout.js

```diff
@@ -149,31 +153,27 @@ export const LocationLayout = ({ location, images }) => {
                 description={location.header_p}
                 img_data={images[location.name]}
             />
-            <Container direction="column">
-                <FirstSection padding="12rem 0">
-                    <Header align="center" as="h2" size={'var(--text-size-header-1)'}>
-                        {`Living in ${display_name}`}
+            <CareerContainer>
+                <FirstSection>
+                    <Header align="center" as="h2" size="var(--text-size-header-5)">
+                        <Localize translate_text="Living in " />
+                        {display_name}
                     </Header>
                     <Flex tablet_direction="column">
                         <Text size="var(--text-size-sm)">{location.first_p}</Text>
                         <ImageWrapper>
                             <QueryImage
                                 data={images[location.first_img]}
-                                alt={`Living in ${display_name}`}
+                                alt={Localize('Living in ') + display_name}
```

Comment:
Does `Localize` work as a function or should it be `localize`?

Reply:
my bad, fixed